ethcore/src/trace/db.rs

```diff
@@ -102,7 +102,10 @@ enum CacheId {
 	Bloom(TraceGroupPosition),
 }
 
-/// Trace database.
+/// Whenever a transaction is executed by evm it's execution trace is stored
+/// in trace database. Each trace has information, which contracts have been
+/// touched, which have been created during the execution of transaction, and
+/// which calls failed.
```

Comment:
multi-line docs should have a single line first, then a gap, and then the rest. it formats much more nicely on the generated docs

Reply:
I've tried, but failed to describe the whole thing in one compact line for a header. Or should I reformat the very same text but with an extra blank line?

On the other hand it wouldn't be great to write meaningless first line that Captain Obvious would be proud of.

Reply:
an obvious line like "Block and transaction execution trace database" gives more information than the struct name while allowing for more information to be included later.